client/src/components/organisms/RequestGroupForm.tsx

```diff
@@ -264,19 +265,26 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
     /* Functions for RequestGroup's Image */
     const updateImageError = (image: string) => {
         let error = "";
-        if (image === "") {
+        if (image === "" && uploadedImg === "") {
```

Comment:
Checking `uploadedImg` here will introduce race conditions since `setUploadedImg` (which is presumably called before calling `updateImageError`) asynchronously updates the value of `uploadedImg`

Since the use of `updateImageError` is `updateImageError(uploadedImg)`, this is probably sufficient:
```
if(image === "") {
  ...
}
```

Reply:
Fixed.